paket-files/haf/YoLo/YoLo.fs

```diff
@@ -622,7 +622,7 @@ module App =
 
   /// Get the assembly resource
   let resource name =
-#if DNXCORE50
+#if NETSTANDARD1_5
```

Comment:
hmm wondering why the file is tracked at all? .gitignore has `paket-files/` entry